cmd/api-mock/main.go

```diff
@@ -36,6 +37,7 @@ func main() {
 		"/api/addons_mgmt/v1/clusters/{cluster_id}/status",
 		NewAddonStatusCreateEndpoint(addonStatusStore),
 	)
+	r.HandleFunc("/configure", configure).Methods("PATCH")
```

Comment:
@pepedocs  this "configure" is to handle and create a mock failure right? can we name it better

Reply:
@venkateshsredhat Nope, that is to configure/patch a live api-mock server. So one can enable/disable a mock api on-the-fly.

Reply:
Ahh okay so you just set the flag there by patching it . okay makes sense